### ISSUE TITLE
vagrant-libvirt provider priority

### ISSUE THREAD

**Issue by pvalena** [vagrant-libvirt/vagrant-libvirt#773]
TL;DR: Could you specify vagrant-libvirt provider priority?

### Steps to reproduce
1. Set up Vagrantfile without specifying providers
2. run `vagrant up` without specifying --provider

### Expected behaviour
Selection of provides should follow provider priority, but vagrant-libvirt has no priority specified.
(Vagrant determines / selects provider based on 'first usable'.)
Please see the issue we are having at
https://bugzilla.redhat.com/show_bug.cgi?id=1444492

### Actual behaviour
The selection of provider follows priority specified by vagrant-libvirt.

### System configuration
**OS/Distro version:**:
Fedora 24

**Libvirt version:**
libvirt-1.3.3.2-1.fc24.x86_64

**Are you using upstream vagrant package or your distros package?**
Distro

**Comment by agriffis** [comment]
Vagrant upstream just merged https://github.com/mitchellh/vagrant/pull/8558 which adds `VAGRANT_PREFERRED_PROVIDERS`. At this point it's not necessary (maybe not even correct) for vagrant-libvirt to internally set a priority. Do you agree @pvalena?

**Comment by pvalena** [comment]
Yes, `VAGRANT_PREFERRED_PROVIDERS` implementation obsoletes this issue.
